leadbutt.py

```diff
@@ -17,6 +19,8 @@
 import datetime
 import os.path
 import sys
+import time
+from retrying import retry
```

Comment:
Thanks for clueing me into `retrying`. I've been lucky enough where I haven't had to retry much in AWS, but it is best practice and it's cool to see this project grow up.

minor style note: since `retrying` this is a third party app, it would go into the second group of imports.